wordpress-mobile/gutenberg-mobile
# [FIX] Cover block background when there's a video is totally black

Fixes #2104 

Gutenberg PR - https://github.com/WordPress/gutenberg/pull/21508

To test:
- Run Wordpress-Android with the packager
- Add a cover block and set video as a background
- the background should be set to the same as a placeholder until the video is loaded

PR submission checklist:
